modules/cluster-logging-rn-5.2.11.adoc

```diff
@@ -0,0 +1,60 @@
+[id="cluster-logging-release-notes-5-2-11"]
+= Logging 5.2.11
+This release includes https://access.redhat.com/errata/RHBA-2022:5012[RHBA-2022:5012-OpenShift Logging Bug Fix Release 5.2.11]
+
+[id="openshift-logging-5-2-11-bug-fixes"]
+== Bug fixes
+* Before this update, clusters configured to perform CloudWatch forwarding wrote rejected log files to temporary storage, causing cluster instability over time. With this update, chunk backup for CloudWatch has been disabled, resolving the issue. (link:https://issues.redhat.com/browse/LOG-2635[LOG-2635])
+
+[id="openshift-logging-5-2-11-CVEs"]
+== CVEs
+.Click to expand CVEs
```

Comment:
How does this look in the rendered version? If I look at other (few) instances in openshift-docs that use `[%collapsible]`, the heading usually says something like `.List of Hive managed resources`.

Reply:
I've used that format for CVEs for a couple months now, renders like this: https://docs.openshift.com/container-platform/4.10/logging/cluster-logging-release-notes.html#openshift-logging-5-4-1-CVEs -- I couldn't find anything in the supplemental style guide, or the openshift docs guidelines that specified any particular phrasing.